
Split the SUNWvbox.pkg rule into two so that files can be installed into
the default layout without building the package file.

diff -wpruN '--exclude=*.orig' a~/src/VBox/Installer/solaris/Makefile.kmk a/src/VBox/Installer/solaris/Makefile.kmk
--- a~/src/VBox/Installer/solaris/Makefile.kmk	1970-01-01 00:00:00
+++ a/src/VBox/Installer/solaris/Makefile.kmk	1970-01-01 00:00:00
@@ -553,7 +553,6 @@
 		$(VBOX_PATH_SOL_INST_SRC)/vbox.pkginfo \
 		$(VBOX_PATH_SOL_INST_SRC)/vbox-ips.mog \
 		\
-		$(if-expr defined(VBOX_WITH_ADDITIONS_PACKING),$(PATH_STAGE_BIN)/additions/VBoxGuestAdditions.iso,) \
 		$(if-expr !defined(VBOX_OSE) && defined(VBOX_WITH_VBOXSDL), $(PATH_DEVTOOLS_TRG)/libsdl/v1.2.13/lib/libSDL-1.2.so.0.11.2,) \
 		\
 		$(if $(VBOX_OSE),,$(foreach arch, x86 amd64, $(foreach lib, libgcc_s.so.1 libstdc++.so.6, \
@@ -614,8 +613,6 @@
 		"$($(var).SRC)/$(file)" "$($(var).DST)/$(file)")))
 
 # VirtualBox: Common files.
-	$(if-expr defined(VBOX_WITH_ADDITIONS_PACKING) \
-	,$(INSTALL) -m 0644 $(PATH_STAGE_BIN)/additions/VBoxGuestAdditions.iso  $(VBOX_PATH_SI_SCRATCH_PKG)/opt/VirtualBox/,)
 	$(LN_SYMLINK) -f ./pkginstall.sh $(VBOX_PATH_SI_SCRATCH_PKG)/opt/VirtualBox/ipsinstall.sh
 	$(SED) -e "s/_HARDENED_/$(if $(VBOX_WITH_HARDENED),hardened,)/" \
 		--output $(VBOX_PATH_SI_SCRATCH_PKG)/opt/VirtualBox/vboxconfig.sh \
@@ -631,9 +628,6 @@
 	$(if-expr defined(VBOX_WITH_QTGUI) \
 	,$(NLTAB)$(LN_SYMLINK) ../rdesktop-vrdp-keymaps/ $(VBOX_PATH_SI_SCRATCH_PKG)/opt/VirtualBox/$(VBOX_SI_ARCH)/rdesktop-vrdp-keymaps,)
 
-	$(if-expr defined(VBOX_WITH_ADDITIONS_PACKING) \
-	,$(NLTAB)$(LN_SYMLINK) ../VBoxGuestAdditions.iso $(VBOX_PATH_SI_SCRATCH_PKG)/opt/VirtualBox/$(VBOX_SI_ARCH)/,)
-
 	$(if-expr !defined(VBOX_OSE) && defined(VBOX_WITH_VBOXSDL) \
 	,$(INSTALL) -s -m 0644 $(PATH_DEVTOOLS_TRG)/libsdl/v1.2.13/lib/libSDL-1.2.so.0.11.2 \
 		$(VBOX_PATH_SI_SCRATCH_PKG)/opt/VirtualBox/$(VBOX_SI_ARCH)/libSDL-1.2.so.0,)
@@ -700,7 +694,7 @@
 # Creates the System V style installer package.
 #
 solaris-package:: $(VBOX_PATH_SI_SCRATCH)/$(PKG_FILENAME).pkg
-$(VBOX_PATH_SI_SCRATCH)/$(PKG_FILENAME).pkg: \
+solaris-install:: \
 		$(VBOX_PATH_SI_SCRATCH)/dist-copy.ts \
 		$(VBOX_SI_DEP_ON_MAKEFILE) \
 		| $$(dir $$@)
